lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr_context.c

```diff
@@ -86,20 +86,21 @@ ebi_t esm_ebr_context_create(
   ue_mm_context_t* ue_mm_context =
       PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(LOG_NAS_ESM, "ESM-PROC  - ue_mm_context null\n ");
+    OAILOG_ERROR_UE(
```

Comment:
ue_mm_context does not exist in this case, let's not change this

Reply:
Resolved